vnext/Shared/Shared.vcxitems

```diff
@@ -4,6 +4,7 @@
     <MSBuildAllProjects>$(MSBuildAllProjects);$(MSBuildThisFileFullPath)</MSBuildAllProjects>
     <HasSharedItems>true</HasSharedItems>
     <ItemsProjectGuid>{2049dbe9-8d13-42c9-ae4b-413ae38fffd0}</ItemsProjectGuid>
+    <ItemsProjectName>Microsoft.React.Shared</ItemsProjectName>
```

Comment:
we should reuse existing terminology - we don't use React anywhere today, so let's call this `Microsoft.ReactNative.Shared` instead

Reply:
Done (reflected on next push).